src/highlighter.js

```diff
@@ -242,6 +244,17 @@ export default function (options) {
             }
             currentNode = childNodes[i];
 
+            if (isBlacklisted(currentNode)) {
+                if (isElement(currentNode)) {
```

Comment:
I made a bug here (my `continue` statement caused the final `break` block to be skipped), so I had to make another change. The resulting diff looks more complicated than it is, here's the high level:
```
if (isBlacklisted(currentNode)) {
    if (isElement(currentNode)) {
        //go deeper in case a descendant of the current blacklisted is whitelisted
        wrapTextNodesInRange(currentNode, rangeInfos);
    }
} else {
    // unchanged code, 75 lines
}
// end wrapping?
```